pages/api/og.tsx

```diff
@@ -7,7 +7,7 @@ export const config = {
 
 // eslint-disable-next-line compat/compat
 const font = fetch(
-  new URL('../../public/fonts/Poppins-Medium.ttf', import.meta.url),
+  new URL('../../public/fonts/Inter-Medium.otf', import.meta.url),
```

Comment:
You can actually remove the fonts directory since we use Google fonts for Inter. https://github.com/livepeer/docs/blob/d1eed0c9ee6a2cd5a214130898f15bf7e018cde8/theme.config.tsx#L163